### PR TITLE
feat(AR-775): watchtower + url handler

### PR DESCRIPTION
**Pull request by daniel-bcw**

### CODE REVIEW

**Review by syed-bcw — APPROVED** [review]
Cool 😎

File: nodejs/watchtower/integration.tests/subscribe.topic.test.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+console.clear();
+
+require('dotenv').config({path: '.env'});
+const urlHandler = require('../../handlers/url.handler');
+const watchtowerHandler = require('../../handlers/watchtower.handler');
+const io = require('socket.io-client');
+let socket;
+let topicSubscriptionPayload;
+const topicTestnet = `3493291`;
```
Comment (aaron-bcw) [inline nodejs/watchtower/integration.tests/subscribe.topic.test.js:9]: Is this going to fail after a testnet reset?

**Review by aaron-bcw — COMMENTED** [review]
Thanks for updating the handlers for other tests 🙏

File: nodejs/watchtower/integration.tests/subscribe.topic.test.js (hunk 1 above)
Comment (syed-bcw) [inline nodejs/watchtower/integration.tests/subscribe.topic.test.js:9]: This test will fail on every testnet reset 🥲

**Review by syed-bcw — APPROVED** [review]
Awesome tests ✅

File: nodejs/watchtower/integration.tests/subscribe.topic.test.js (hunk 1 above)
Comment (syed-bcw) [reply nodejs/watchtower/integration.tests/subscribe.topic.test.js:9]: Yes 🥲

File: nodejs/.env-example [hunk 2]
```diff
@@ -8,4 +8,10 @@ COMMUNITY_TESTNET_URL=https://testnet.hashio.io/api
...
 
 TESTNET_ACCOUNT_ID=0.0.1417
-TESTNET_ACCOUNT_ID_EVM=0x0000000000000000000000000000000000000589
\ No newline at end of file
+TESTNET_ACCOUNT_ID_EVM=0x0000000000000000000000000000000000000589
+
+ARKHIA_MAINNET_WATCHTOWER=
+ARKHIA_TESTNET_WATCHTOWER=
+
+TESTNET_TOPIC_ID=3493291
```
Comment (syed-bcw) [inline nodejs/.env-example:16]: 👍

**Pull request merged**